homeassistant/components/synology_dsm/sensor.py

```diff
@@ -102,7 +103,7 @@ class SynoDSMUtilSensor(SynoDSMSensor, SensorEntity):
     """Representation a Synology Utilisation sensor."""
 
     @property
-    def state(self):
+    def state(self) -> Any | None:
```

Comment:
It'd be nice to list all types `attr` can have. E.g. `int | float | None`.

Reply:
would like to use `StateType` from `homeassistant.helpers.typing`, but still need to type hinting underlying python lib first :confused:

Reply:
It's fine. Let's keep this for now 🙂